### ISSUE TITLE
Write scripts to market make the pools on testnet

### ISSUE THREAD

**Issue by Paul4912** [tracer-protocol/perpetual-pools-contracts#271]
Write scripts to populate long and short sides of the pools. Need to do this to verify poolsv2 works and to get ready for others to use our testnet deployment.

End result is:
- A reusable script that populates both sides of a newly-deployed testnet market with settlement tokens.
- Testnet v2 deployments are all populated on long and short

# Steps

- [x] Commit long and short script for one pool
- [ ] Upkeep script on one pool
- [ ] Generalise to work for array of pools
- [ ] Run them all on the Pools v2 deployments

**Comment by nisdo** [comment]
Hi @Paul4912 if this is blocked can you add the comment here on why so we can set framing for how we plan to unblock it (and link to any Github issues that may be needed to track the work there to unblock)?

**Comment by nisdo** [comment]
There are steps still incomplete in this task list so it should not yet be close @mynameuhh @Paul4912 unless it is being marked as Duplicate or superceeded by another issue which is linked. As next steps please include the issues linked to any coding tasks above that require review, and add these individually to the sprint so we can track the child tasks and then only close the parent issues when all the child issues are completed and checked off.

**Comment by Paul4912** [comment]
Committed work in progress to 
https://github.com/tracer-protocol/perpetual-pools-contracts/tree/mmscripts
before i go on leave.

Currently running into an issue when upkeep.ts. Commits seem to be working fine but upkeepsinglepool failing.

Here is output

<img width="586" alt="Screen Shot 2021-12-17 at 12 48 06 pm" src="https://user-images.githubusercontent.com/53968911/146475316-3e6b2be0-f41f-4d16-a642-0b752e813484.png">

**Comment by mynameuhh** [comment]
@ScaredIbis
